no_drama/__main__.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+
+import argparse
+import tempfile
+import os
+import os.path
+import shutil
+import subprocess
+import zipfile
+import glob
+import fnmatch
+
+from string import Template
+from functools import partial
+
+
+this_file = os.path.realpath(__file__)
+this_directory = os.path.dirname(this_file)
+templates_dir = os.path.join(this_directory, 'templates')
+build_lib = os.path.join(this_directory, 'build_lib')
+build_files = os.path.join(this_directory, 'build_files')
+
+
+def save_wheels(destination, packages=None, requirements_file=None,
+                find_links=None):
+    call_list = ["pip", "wheel", "--wheel-dir=%s" % destination]
+
+    if packages is not None:
```

Comment:
It will check for `not None`, as well as other possible cases that fail a boolean where we'd either want to handle it differently or know why `packages` was a completely unexpected type. The potential for unintended side-effects suggests to me that leaving out explicit `is [not] None` in all of these cases is not simpler or cleaner.

[PEP8 also warns against test `if not foo` as opposed to `if foo is not None`.](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)